{
  try {
    backupService().doIncrementalBackupOrFallbackToFull(BACKUP_HOST,56789,backupDir,ConsistencyCheck.NONE,dbRule.getConfigCopy(),BackupClient.BIG_READ_TIMEOUT,false);
    fail("No exception thrown");
  }
 catch (  RuntimeException e) {
    assertThat(e.getMessage(),containsString("BackupClient could not connect"));
    assertThat(e.getCause(),instanceOf(ConnectException.class));
  }
}
